src/_scss/pages/covid19/_budgetCategoriesTable.scss

```diff
@@ -15,6 +15,58 @@
             }
         }
     }
+    
+    .usda-table {
+        margin-top: 0;
+        td {
+            button {
+                float: left;
+                margin-top: rem(5) !important;
```

Comment:
Why are we doing `!important` here? We shouldn't do this IMO. We should be able to override the component library styles or w/e relatively easily. Let me know if you need some help w/ this.